includes/class-genesis-tabs-widget.php

```diff
@@ -0,0 +1,422 @@
+<?php
+/**
+ * Genesis Tabs Widget file.
+ *
+ * @package genesis-tabs
+ */
+
+/**
+ * Widget Class. Handles the widget form and output.
+ *
+ * @package Genesis_Tabs
+ * @since 0.9.0
+ */
+class Genesis_Tabs_Widget extends WP_Widget {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$widget_ops  = array(
+			'classname'   => 'ui-tabs',
+			'description' => __( 'Displays featured posts in Tabs', 'genesis' ),
+		);
+		$control_ops = array(
+			'width'   => 505,
+			'height'  => 350,
+			'id_base' => 'tabs',
+		);
+		parent::__construct( 'tabs', __( 'Featured Tabs', 'genesis' ), $widget_ops, $control_ops );
+	}
+
+	/**
+	 * Widget Output
+	 *
+	 * @param array $args Args.
+	 * @param array $instance Instance.
+	 */
+	public function widget( $args, $instance ) {
+		extract( $args );
+
+		/* defaults */
```

Comment:
Inline comments should be double-slash, capitalized, period at the end. Semi-sentence structure.
```
// The comment.
$some_variable = 1;
```